sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -543,53 +553,61 @@ def test_unsupervised_radius_neighbors(
                 i1[:] = i1[j]
             results.append((dist, ind))
 
-            assert_array_almost_equal(
-                np.concatenate(list(ind)), np.concatenate(list(ind1))
-            )
+            assert_allclose(np.concatenate(list(ind)), np.concatenate(list(ind1)))
```

Comment:
Not sure what's going on here. Isn't it equivalent to ``assert_array_equal(ind, ind1)`` ?
```suggestion
            assert_array_equal(np.concatenate(list(ind)), np.concatenate(list(ind1)))
```

Reply:
That's because `ind` and `ind1` here are [ragged-array](https://en.wikipedia.org/wiki/Jagged_array). This totally flattens it so that this can be compared easily.